crates/plex-api/src/media_container/server/library.rs

```diff
@@ -0,0 +1,496 @@
+use std::{fmt, str::FromStr};
+
+use monostate::MustBe;
+use serde::{
+    de::{self, Visitor},
+    Deserialize, Deserializer,
+};
+use serde_aux::prelude::{deserialize_number_from_string, deserialize_option_number_from_string};
```

Comment:
There's [serde_with::rust::display_fromstr](https://docs.rs/serde_with/1.14.0/serde_with/rust/display_fromstr/index.html) already included as the dependency. Is there a nice and easy way in Rust/serde to make an existing deserialized return Option?

Reply:
Ok then. I had serde_aux included previously anyway :)